### ISSUE TITLE
continue training with .caffemodel

### ISSUE THREAD

**Issue by whansk50** [rbgirshick/py-faster-rcnn#950]
Hi
I'm using this codes with Python3.8 (with modifying some codes that can execute on Python3, because code was made with 2.7)
Everything's okay, but some errors appear while training.
Fixing codes spend not much time, but (maybe) cause of using single gpu, it takes too much time.
Moreover I only knew command from README that execute training from the beginning (./experiments/scripts/faster_rcnn_end2end(or alt_opt).sh ~~~) so just did that command regardless of getting iterations 70,000 at previous training, again and again unwillingly.

Well, I could see the caffemodel file (I think it is saved when iterations got 10,000 times) in directory following README, so what I want to know is :

Can I continue training with previous stopped iterations using caffemodel file or other files? If I can, what command is that?

Thanks for reading with my poor english

**Comment by karandeepdps** [comment]
Yes, you can resume the training from a previous checkpoint. A checkpoint includes not only the model architecture and weights (the .caffemodel file) but also the state of the optimizer, the epoch, the learning rate scheduler, and so on.
